maquetado/scss/books.scss

```diff
@@ -43,7 +45,10 @@
 }
 
 .book-info-container {
+  display: grid;
   flex-grow: 1;
+  grid-template-columns: 2fr 3fr;
+  grid-template-rows: auto 1fr 1fr 1fr 1fr 1fr 1fr;
```

Comment:
pretty sure you can do `grid-template-rows: auto repeat(6, 1fr);`

https://developer.mozilla.org/en-US/docs/Web/CSS/repeat#CSS